docs/getting_started.md

```diff
@@ -1,24 +1,24 @@
 # Memory Analysis Cube
 
-The Memory Analysis Cube (MAC) is an ActivePivot project that aims to provide
+The Memory Analysis Cube (MAC) is an Atoti project that aims to provide
 the necessary tools to explore and analyze the data consumption from any other
-ActivePivot application, starting from ActivePivot 5.8 onward.
+Atoti Server application, starting from Atoti Server 5.8 onward.
 
 The project is built as a standalone Spring Boot jar file with an embedded
-ActiveUI application available on `localhost:9092` by default.
+Atoti UI application available on `localhost:9092` by default.
 
 ## Main Features
 
 * Off Heap memory usage monitoring 
 * Store/Field-related memory footprint
 * Cube-related memory footprint
 * Overview of structure-related memory footprint
-* Loading and comparing several ActivePivot applications' exported memory
+* Loading and comparing several Atoti Server applications' exported memory
```

Comment:
```suggestion
* Loading and comparing several Atoti applications' exported memory
```

Reply:
This doc really seems duplicated. I will review it to clean the duplications in another PR.